docs/source/config.json

```diff
@@ -62,9 +62,17 @@
     "API Reference": {
       "ApolloServer": "/api/apollo-server",
       "startStandaloneServer": "/api/standalone",
-      "expressMiddleware": "/api/express-middleware",
-      "@apollo/subgraph": "https://www.apollographql.com/docs/federation/api/apollo-subgraph/",
-      "@apollo/gateway": "https://www.apollographql.com/docs/federation/api/apollo-gateway/"
+      "expressMiddleware": "/api/express-middleware"
+    },
+    "Using with Federation": {
```

Comment:
If we're happy with this being near the bottom anyway (ie, we don't want it up high to be more visible on purpose), let's put it after the plugins documentation, which is actually pretty important docs (lots and lots of configurable options there).